src/auth/.gcb/builds/api_key_test/main.tf

```diff
@@ -0,0 +1,78 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+variable "project" {}
+
+data "google_project" "project" {
+}
+
+# The API Keys service is used to create an API key
+resource "google_project_service" "apikeys" {
+  project = var.project
+  service = "apikeys.googleapis.com"
+
+  timeouts {
+    create = "30m"
+    update = "40m"
+  }
+
+  disable_dependent_services = true
+}
+
+# The language service is used to verify the API key
+resource "google_project_service" "language" {
+  project = var.project
+  service = "language.googleapis.com"
+
+  timeouts {
+    create = "30m"
+    update = "40m"
+  }
+
+  disable_dependent_services = true
+}
+
+resource "google_apikeys_key" "test-api-key" {
```

Comment:
Does this mean that the key value is stored in the Terraform bucket?  Just curious, I am not sure it changes anything.

Reply:
FWIW, the value of all of the secrets are stored in the terraform bucket.